README.md

```diff
@@ -92,3 +96,78 @@ npm install
 npm run start
 ```
 
+## Docker:
+
+The provided docker-compose file allows you to deploy Redis, Postgres and the Alephium mining pool in 3 simple steps:
+
+
+### Clone the repo and move to its folder:
+```shell
+git clone https://github.com/alephium/mining-pool.git && cd mining-pool
+```
+
+### Build the Docker image:
+
+1. From local sources:
+```shell
+docker build -t mining-pool:latest .
```

Comment:
with the `build:` section inside of compose, you can make this automatic :+1: 

It will name and tag what it builds locally with what you place under `image: name:tag` 

https://docs.docker.com/compose/compose-file/build/

Reply:
Thanks!

I added the step and adjusted the readme